src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -177,10 +181,20 @@ private static List<Species> CopyInitialPopulationsToResults(SimulationConfigura
                 var totalNicheFitness = 0.0f;
                 foreach (var currentSpecies in species)
                 {
-                    // Softly enforces https://en.wikipedia.org/wiki/Competitive_exclusion_principle
-                    // by exaggerating fitness differences
-                    var thisSpeciesFitness =
-                        Mathf.Max(Mathf.Pow(niche.FitnessScore(currentSpecies, cache), 2.5f), 0.0f);
+                    float thisSpeciesFitness;
+
+                    if (strictCompetition)
+                    {
+                        // Softly enforces https://en.wikipedia.org/wiki/Competitive_exclusion_principle
+                        // by exaggerating fitness differences
+                        thisSpeciesFitness =
+                            Mathf.Max(Mathf.Pow(niche.FitnessScore(currentSpecies, cache), 2.5f), 0.0f);
+                    }
+                    else
+                    {
+                        thisSpeciesFitness = Mathf.Max(niche.FitnessScore(currentSpecies, cache), 0.0f);
```

Comment:
I am pretty confident that this would backfire VERY badly. My guess is that the single most important part of the auto-evo refactor was adding the exclusion principle, which allows for a diversity of species instead of one all-rounder dominating the field. If a lot of testing proves otherwise I'm happy to re-evaluate, but otherwise if you take no other suggestions from this review multiply this by an exponent meaningfully greater than one. Doing otherwise is very likely to result in a very undynamic environment of NPC species.

Reply:
The reason I made this change is due to players experiencing that adding basically *any* organelle just saps their population. So I thought that if the competition was less harsh, then adding an organelle that is not very heavily competed, *should* be better.
So as the branch says, this is an emergency change. Though, if this actually makes the gameplay experience worse, I'll disable it before merging.

Reply:
The problem is that the AI is trying on random organelles far more often than the player does, so if that's NOT being punished I suspect you will find the "one of everything" species we saw in 0.5.4. Is there any chance I could convince you to pull just this change out to a separate PR, to be tested independently? I'm certainly interested in double-checking my assumptions, but I don't want to conflate variables with the other changes being done here, which I suspect are much safer.